grpc-gateway/pb/updateDeviceMetadata.go

```diff
@@ -0,0 +1,38 @@
+package pb
+
+import (
+	"context"
+
+	"github.com/gofrs/uuid"
+	"github.com/plgd-dev/cloud/resource-aggregate/commands"
+	"google.golang.org/grpc/peer"
+)
+
+func (req *UpdateDeviceMetadataRequest) ToRACommand(ctx context.Context) (*commands.UpdateDeviceMetadataRequest, error) {
+	correlationUUID, err := uuid.NewV4()
+	if err != nil {
+		return nil, err
+	}
+	connectionID := ""
+	peer, ok := peer.FromContext(ctx)
+	if ok {
+		connectionID = peer.Addr.String()
```

Comment:
in case `ok == false` we create a `CommandMetadata` object with empty `ConnectionId`, is that OK?

Reply:
Yes. it's ok. We just fill it in when we have such information.